renderer/src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,11 +1,11 @@
 import { useState } from 'react'
-import HeaderBackgroundImage from '../assets/img/header.png'
-import Modal from '../components/Modal'
-import ActivityLog from '../components/ActivityLog'
-import UpdateBanner from '../components/UpdateBanner'
-import WalletWidget from '../components/WalletWidget'
-import useStationActivity from '../hooks/StationActivity'
-import { formatTokenValue } from '../utils/number-ops'
+import HeaderBackgroundImage from 'src/assets/img/header.png'
```

Comment:
I am not a fan of letting TypeScript rewrite the import paths. When I look at this import, I need to recall that `src` is mapped in TypeScript config before I can figure out where to find the imported file. I find relative import path easier to reason about.

I guess this is not a big deal as long as IDEs like VS Code understand this TypeScript feature and can navigate around the project when using commands like "go to definition".

I am not actively working on the front-end, so you should not put too much weight to my opinions. Choose what works best for you.

Reply:
It makes nested files easier to work with, and as long as the rewritten path matches the actual file structure inside "src", I think it remains clear enough.